src/utils/Sort.js

```diff
@@ -145,8 +145,8 @@ export function GetColorCategory(type, colors) {
 
 export const SORTS = [
   'Artist',
-  'CMC',
-  'CMC2',
+  'Mana Value',
```

Comment:
I'm glad you went with 'Mana Value' and not "MV" and "MV2"

Reply:
Yeah, I'm not a fan of using "MV" if it isn't necessary.